app/controllers/StartController.scala

```diff
@@ -25,6 +25,6 @@ import views.html.start_page
 class StartController @Inject()(mcc: MessagesControllerComponents, startPage: start_page) extends FrontendController(mcc) with I18nSupport {
```

Comment:
Do we still need the start_page view? Will it maybe removed in future?

Reply:
Yes, next ticket 2965 we will remove all links to the start page and remove the controller and view then.

Reply:
OK, but do we still need to inject `startPage`? It's not used anywhere.

Reply:
Just ended up pulling the whole start_page view in the end :+1: